common/lib/capa/capa/responsetypes.py

```diff
@@ -583,7 +582,7 @@ def _render_response_msg_html(self, response_msg):
         # First try wrapping the text in a <div> and parsing
         # it as an XHTML tree
         try:
-            response_msg_div = etree.XML('<div>%s</div>' % str(response_msg))
+            response_msg_div = etree.XML('<div>{}</div>'.format(str(response_msg)))  # xss-lint: disable=python-wrap-html
```

Comment:
@dianakhuang: After you made the other changes, would this work using the two calls to`HTML`?  Not a big deal, but it would make it more clear that you are expecting `response_msg` to be HTML.